cob_omni_drive_controller/src/param_parser.cpp

```diff
@@ -112,7 +160,12 @@ bool parseWheelGeom(WheelGeom & geom, XmlRpc::XmlRpcValue &wheel, MergedXmlRpcSt
     if(model && !geom.steer_name.empty()){
         steer_joint = model->getJoint(geom.steer_name);
         if(steer_joint){
-            steer_pos = steer_joint->parent_to_joint_origin_transform.position;
+            tf2::Transform transform;
+            if(parseWheelTransform(geom.steer_name, "base_link", transform, model)){
```

Comment:
link names should be hard-coded!

Reply:
hmm, I know. Because the original version also had this had-coded link name I thought I keep it that way ;)

Reply:
>  Because the original version also had this had-coded link name I thought I keep it that way

Which version? The plugins here are meant to be generic, they do not include hard-coded names.